host/bcm283x/gpio.go

```diff
@@ -326,12 +329,17 @@ func (p *Pin) FastOut(l gpio.Level) {
 // PWM pins
 //
 // PWM0 is exposed on pins 12, 18 and 40.
+// However, PWM0 is used for generating clock for DMA and unavailable for PWM.
+// TODO(simokawa): Use other DREQ source like PCM.
 //
 // PWM1 is exposed on pins 13, 19, 41 and 45.
 //
 // PWM0 and PWM1 share the same 25Mhz clock source. The period must be a
 // divisor of 25Mhz.
 //
+// DMA driven PWM is aviable for all pins except PWM1 pins, its resolution is
```

Comment:
Let's commit the working code without user visible changes first, then the user visible change afterwards.

Reply:
I reverted this file for this change.